main/src/hadd.cxx

```diff
@@ -893,16 +931,12 @@ int main(int argc, char **argv)
 #endif
 
    if (status) {
-      if (verbosity == 1) {
-         Info() << "merged " << allSubfiles.size() << " (" << fileMerger.GetMergeList()->GetEntries()
-                << ") input (partial) files into " << targetname << ".\n";
-      }
+      Info() << "merged " << allSubfiles.size() << " (" << fileMerger.GetMergeList()->GetEntries()
+             << ") input (partial) files into " << targetname << "\n";
       return 0;
    } else {
-      if (verbosity == 1) {
-         Err() << "failure during the merge of " << allSubfiles.size() << " ("
-               << fileMerger.GetMergeList()->GetEntries() << ") input (partial) files into " << targetname << ".\n";
-      }
+      Err() << "failure during the merge of " << allSubfiles.size() << " (" << fileMerger.GetMergeList()->GetEntries()
```

Comment:
I assume the message was previously only optionally printed because it was assume that the main cause of failure would already have been printed message and this was 'only' extra details about the setups.

Unrelated to this PR, the message could be improve to clarify what the 2 numbers means.